src/index.ts

```diff
@@ -440,21 +439,23 @@ const main = async (baseInfo: LSPluginBaseInfo) => {
       type: 'string',
       title: 'Enter the template to use for new articles',
       description:
-        'Enter the template to use for new articles. Required variables are: {title}, {omnivoreUrl}. Optional variables are: {siteName}, {originalUrl}, {author}, {labels}, {dateSaved}',
-      default: `[{title}]({omnivoreUrl})
+        'Enter the template to use for new articles. Required variables are: {{{title}}}, {{{omnivoreUrl}}}. Optional variables are: {{{siteName}}}, {{{originalUrl}}}, {{{author}}}, {{{labels}}}, {{{dateSaved}}}',
+      default: `[{{{title}}}]({{{omnivoreUrl}}})
       collapsed:: true
-      site:: [{siteName}]({originalUrl})
-      author:: {author}
-      labels:: {labels}
-      date_saved:: {dateSaved}`,
+      site:: [{{{siteName}}}]({{{originalUrl}}})
+      author:: {{{author}}}
+      {{#labels.length}}
+      labels:: {{#labels}}[[{{{name}}}]]{{/labels}}
+      {{/labels.length}}
```

Comment:
@jacksonh this code will check the size of labels and prevent showing an empty list of labels on the page